Baking Converter/ConverterViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  ConverterViewController.swift
+//  Baking Converter
+//
+//  Created by Michael Johnson on 6/10/17.
+//  Copyright © 2017 Michael Johnson. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+class ConverterViewController: UIViewController {
+
+    @IBOutlet weak var ingredientsPicker: UIPickerView!
+    @IBOutlet weak var inputUnitsPicker: UIPickerView!
+    @IBOutlet weak var outputUnitsPicker: UIPickerView!
+    @IBOutlet weak var inputField: UITextField!
+    @IBOutlet weak var resultsLabel: UILabel!
+    
+    var controller: ConverterController!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Configure results label with rounded corners
+        resultsLabel.layer.cornerRadius = 8.0
+        resultsLabel.layer.masksToBounds = true
+        controller = ConverterController()
+    }
+    
+    @IBAction func textFieldEditingChangedAction(_ sender: Any) {
+        controller.converterSceneInputTextDidChange(self)
+    }
+    
+}
+
+extension ConverterViewController: UIPickerViewDataSource {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        switch pickerView {
+        case inputUnitsPicker:
+            return controller.numberOfInputUnitsOptions
+        case outputUnitsPicker:
+            return controller.numberOfOutputUnitsOptions
+        default:
```

Comment:
Whenever I have a default like this, I put a comment of what case that would cover. Also, I believe that Swift 3 allows you to cover all cases and not need a default.

Reply:
I changed the switch statements so that they're more consistent with each other: each now has a `default` case which shouldn't ever be reached, and a comment indicating that. I need some sort of `default` clause for these because I'm switching on the identify of a `UIPickerView` reference, so I can't cover it exahustively. But I think it's a little clearer like this, rather than how I had it before.